uasc/secure_channel.go

```diff
@@ -38,6 +39,16 @@ type Response struct {
 }
 
 type SecureChannel struct {
+	// SecureChannelID is a unique identifier for the SecureChannel assigned by the Server.
```

Comment:
nit: move this to the other unexported fields

Reply:
done